packages/features/insights/context/FiltersProvider.tsx

```diff
@@ -16,9 +18,14 @@ const querySchema = z.object({
   userId: z.coerce.number().nullable(),
   memberUserId: z.coerce.number().nullable(),
   eventTypeId: z.coerce.number().nullable(),
-  filter: z.enum(["event-type", "user"]).nullable(),
+  filter: z
```

Comment:
Add routing_form and bookingstatus as valid options 

Also allow rf_* for any dynamic fields that come from the routing form specifcially